server/belga/search_providers.py

```diff
@@ -331,6 +327,33 @@ def __init__(self, provider):
     def url(self, resource):
         return urljoin(self.base_url, resource.lstrip("/"))
 
+    def get_search_text(self, query):
+        try:
+            searchText = query["query"]["filtered"]["query"]["query_string"]["query"]
+        except KeyError:
+            searchText = ""
+
+        return searchText
+
+    def get_highlight(self, search_text, docs):
```

Comment:
should be probably called rather `set_highlight` as it's setting it and not returning anything